torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -645,7 +645,32 @@ def rotate(
         return rotate_image_pil(inpt, angle, interpolation=interpolation, expand=expand, fill=fill, center=center)
 
 
-pad_image_pil = _FP.pad
+def _parse_pad_padding(padding: Union[int, List[int]]) -> List[int]:
+    if not isinstance(padding, (int, tuple, list)):
+        raise TypeError(f"`padding` should be an integer or tuple or list of integers, but got {padding}")
```

Comment:
Nit: Shall we move to the if block below where we can first check for int, then tuple/list, else error?